import/popups/PopupOpener.qml

```diff
@@ -99,28 +99,28 @@ Timer {
         }
         case "embed:permissions": {
             if (data.title === "geolocation"
-                    && Popups.LocationSettings.enabled
-                    && Popups.LocationSettings.gpsPowered) {
+                    && Popups.LocationSettings.locationEnabled
+                    && Popups.LocationSettings.gpsEnabled) {
                 switch (Geolocation.response(data.host)) {
                     case "accepted": {
-                        contentItem.sendAsyncMessage("embedui:premissions",
+                        contentItem.sendAsyncMessage("embedui:permissions",
```

Comment:
Nice one, typo exists also over here: 

https://git.merproject.org/mer-core/embedlite-components/blob/master/jscomps/ContentPermissionPrompt.js#L114

https://git.merproject.org/mer-core/embedlite-components/blob/master/jscomps/ContentPermissionPrompt.js#L63

Could you prepare a separate PR about these typo fixes if this ends up being declined?

Regarding startUpdates, it's called over here.

https://git.merproject.org/mer-core/gecko-dev/blob/master/dom/system/qt/QTMLocationProvider.cpp#L66